tests/testgroup80.py

```diff
@@ -412,6 +407,54 @@ def runTest(self):
                 self.dataplane.send(of_ports[0], str(pkt))
                 receive_pkt_check(self.dataplane, pkt,[yes_ports], no_ports, self)
 
+class Grp80No250(base_tests.SimpleDataPlane):
+
+    """Verify switch is follow the value of miss_send_len in OFPT_GET_CONFIG_REPLY message """
+    @wireshark_capture
```

Comment:
This line be removed as now we have wireshark captures defined in `setUP()` method of parent class.
@jonstout @ytch